docs/modules/utils/examples/gmail.ipynb

```diff
@@ -0,0 +1,90 @@
+{
```

Comment:
This should go in https://github.com/hwchase17/langchain/tree/master/docs/modules/agents/toolkits/examples and follow the same examples there